semcore/utils/src/use/useFocusLock.ts

```diff
@@ -136,7 +140,7 @@ const useFocusLockHook = (
   focusMaster = false,
   onFocusOut?: (event: Event) => void,
 ) => {
-  useFocusBorders(React, disabled);
+  useFocusBorders(React, trapRef, disabled);
```

Comment:
Let's add it as a third attribute (just to make focus lock merging safer if it will happen anywhere)

Reply:
fixed